checks.d/postgres.py

```diff
@@ -5,6 +5,10 @@
 # stdlib
 import socket
 
+# project
+from checks import AgentCheck, CheckException
```

Comment:
The CI is failing on the import order and this line being imported twice (L13 is the same). Can you fix that please?